src/harness/sas_interface.py

```diff
@@ -230,6 +230,17 @@ def BlacklistByFccId(self, request):
     """
     pass
 
+  @abc.abstractmethod
+  def BlacklistDPByFRN(self, request):
```

Comment:
I don't think this is how cert blacklisting will work in normal operation. If it is, can you please provide some information that I can relay to the rest of test dev to explain this addition to the testing API?

My understanding is that we will use OCSP in normal operation and that the certs need to be blacklisted in that way.